recipes/lunr/meta.yaml

```diff
@@ -0,0 +1,79 @@
+{% set name = "lunr" %}
+{% set version = "0.5.4" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+    sha256: 185254a6776d0d1758531dab32823cdbb8327be0158731dbd55b6943f2062387
+    folder: dist
+  - url: https://github.com/yeraydiazdiaz/lunr.py/archive/v{{ version }}.tar.gz
+    sha256: cb2cec0875970f680d8a20399052890acb3d7d18b769078471fabb2a00a80646
+    folder: src
+
+build:
+  number: 0
+  script:
+    - cd dist
+    - {{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - enum34  # [py<34]
+    - future >=0.16.0
+    - python
+    - six >=1.11.0
+test:
+  source_files:
+    - src/tests
+  requires:
+    - mock
+    - nltk >=3.2.5
+    - nodejs >8,<9
+    - pytest
+    - pytest-timeout
+  imports:
+    - lunr
+    - lunr.builder
+    - lunr.exceptions
+    - lunr.field_ref
+    - lunr.idf
+    - lunr.index
+    - lunr.languages
+    - lunr.languages.stemmer
+    - lunr.languages.trimmer
+    - lunr.match_data
+    - lunr.pipeline
+    - lunr.query
+    - lunr.query_lexer
+    - lunr.query_parser
+    - lunr.stemmer
+    - lunr.stop_word_filter
+    - lunr.token
+    - lunr.token_set
+    - lunr.token_set_builder
+    - lunr.tokenizer
+    - lunr.trimmer
+    - lunr.utils
+    - lunr.vector
+
+about:
+  home: https://github.com/yeraydiazdiaz/lunr.py
+  license: MIT
+  license_family: MIT
+  license_file: dist/LICENSE
+  summary: A Python implementation of Lunr.js
+  doc_url: https://lunr.readthedocs.io
+  description: >
```

Comment:
Minor thing but a pipe (`|`) here for line continuation might be better for rendering the text in the final feedstock.